concrete-core-fixture/src/fixture/glwe_ciphertext_vector_decryption.rs

```diff
@@ -0,0 +1,169 @@
+};
+
+use crate::fixture::Fixture;
+use crate::generation::prototyping::{
+    PrototypesGlweCiphertextVector, PrototypesGlweSecretKey, PrototypesPlaintextVector,
+};
+use crate::generation::synthesizing::{
+    SynthesizesGlweCiphertextVector, SynthesizesGlweSecretKey, SynthesizesPlaintextVector,
+};
+use crate::generation::{IntegerPrecision, Maker};
+use crate::raw::generation::RawUnsignedIntegers;
+use crate::raw::statistical_test::assert_noise_distribution;
+
+/// A fixture for the types implementing the `GlweCiphertextVectorDecryptionEngine` trait.
+pub struct GlweCiphertextVectorDecryptionFixture;
+
+#[derive(Debug)]
+pub struct GlweCiphertextVectorDecryptionParameters {
+    pub glwe_dimension: GlweDimension,
+    pub polynomial_size: PolynomialSize,
+    pub count: GlweCiphertextCount,
+    pub noise: Variance,
+}
+
+impl<Precision, Engine, PlaintextVector, SecretKey, CiphertextVector>
+    Fixture<Precision, Engine, (PlaintextVector, SecretKey, CiphertextVector)>
+    for GlweCiphertextVectorDecryptionFixture
+where
+    Precision: IntegerPrecision,
+    Engine: GlweCiphertextVectorDecryptionEngine<SecretKey, CiphertextVector, PlaintextVector>,
+    PlaintextVector: PlaintextVectorEntity,
+    SecretKey: GlweSecretKeyEntity,
+    CiphertextVector: GlweCiphertextVectorEntity<KeyDistribution = SecretKey::KeyDistribution>,
+    Maker: SynthesizesPlaintextVector<Precision, PlaintextVector>
+        + SynthesizesGlweSecretKey<Precision, SecretKey>
+        + SynthesizesGlweCiphertextVector<Precision, CiphertextVector>,
+{
+    type Parameters = GlweCiphertextVectorDecryptionParameters;
+    type RepetitionPrototypes = (
+        <Maker as PrototypesGlweSecretKey<Precision, SecretKey::KeyDistribution>>::GlweSecretKeyProto,
+    );
+    type SamplePrototypes = (
+        <Maker as PrototypesPlaintextVector<Precision>>::PlaintextVectorProto,
+        <Maker as PrototypesGlweCiphertextVector<Precision,
+            SecretKey::KeyDistribution>>::GlweCiphertextVectorProto,
+    );
+    type PreExecutionContext = (SecretKey, CiphertextVector);
+    type PostExecutionContext = (SecretKey, PlaintextVector, CiphertextVector);
+    type Criteria = (Variance,);
+    type Outcome = (Vec<Precision::Raw>, Vec<Precision::Raw>);
+
+    fn generate_parameters_iterator() -> Box<dyn Iterator<Item = Self::Parameters>> {
+        Box::new(
+            vec![
+                GlweCiphertextVectorDecryptionParameters {
+                    glwe_dimension: GlweDimension(200),
+                    polynomial_size: PolynomialSize(256),
+                    count: GlweCiphertextCount(10),
+                    noise: Variance(0.00000001),
+                },
+                GlweCiphertextVectorDecryptionParameters {
+                    glwe_dimension: GlweDimension(1),
+                    polynomial_size: PolynomialSize(2),
+                    count: GlweCiphertextCount(1),
+                    noise: Variance(0.00000001),
+                },
```

Comment:
Is there a reason that the cases `glwe_dimension=1` and `count=1` are tested together here, but separately in the corresponding encryption fixture?

Reply:
@blowfish880 here I reduced the amount of tests just for performance reasons. I think it's ok to make the change for the encryption test so that they're the same. What do you think?

Reply:
@blowfish880 I just pushed the change :smiley: